src/core/Table/TableRowMemoized.tsx

```diff
@@ -80,13 +121,27 @@ const TableRow = <T extends Record<string, unknown>>(props: {
           onClick?.(event, row);
         }}
       >
-        {row.cells.map((cell) => {
+        {row.cells.map((cell, index) => {
           const cellProps = cell.getCellProps({
             className: cx('iui-cell', cell.column.cellClassName),
```

Comment:
Unrelated but we need to provide a way for the user to set cell class name.

Reply:
We have: just pass `cellClassName` property to column.

Reply:
That applies the same class name to all cells in the column. What if there is a need to customize per row?

Reply:
You could use `rowProps` to pass custom class that has `nth-child` selector.